### ISSUE TITLE
How can I get access to Pro features after buying a license?

### ISSUE THREAD

**Issue by thomasdc** [coreui/coreui#16]
My company bought a CoreUI Pro Angular license.
How can we access the Pro features? 
Is there an Pro example project similar to the free version?

Thanks!

**Comment by mrholek** [comment]
Yes. Access to private github repository is available if you have active membership plan

**Comment by mrholek** [comment]
You should have following files:
![zrzut ekranu 2018-06-25 o 13 32 04](https://user-images.githubusercontent.com/3524167/41848012-538d3e3e-787c-11e8-88ae-1f176399e077.png)

Full project = coreui-pro-angular-admin-template-2.0.0

**Comment by thomasdc** [comment]
I didn't make the purchase myself. The person that did sent me a license key. Can you resend me the zip's if I provide you our license key? Do you have a support email address I can mail to?

**Comment by mrholek** [comment]
Send me email with license id to lukasz@coreui.io

**Comment by thomasdc** [comment]
Thanks for the help!

**Comment by cAstraea** [comment]
Did you find out what you have to do ? I was looking for some login page on coreui website but couldn't find anything. Very confusing. I'm in the same situation as you I only have the license key.

**Comment by thomasdc** [comment]
Try contacting Lukasz via his support page: https://coreui.io/support/

**Comment by rlewis86** [comment]
For anyone else having this issue: Deal directly with [support@gumroad.com](mailto:support@gumroad.com).

I tried emailing @mrholek directly as well as [support@coreui.io](mailto:support@coreui.io) twice. No answer from them after 10 days of waiting. I would give them the benefit of the doubt that their email was blocked by our corporate filter, but when I put in for support, the auto-generated email from their domain came through instantly.

Needless to say, I am thoroughly disappointed that the only way we got support is when we asked for a refund. Gives me no hope for future support down the road.
